cpanfile

```diff
@@ -106,6 +106,7 @@ on 'test' => sub {
   requires 'Test::Output';
   requires 'Test::Pod';
   requires 'Test::Warnings';
+  requires 'XML::SemanticDiff';
```

Comment:
I think you will need to add it to docker/travis_test/Dockerfile

Reply:
fixed